tests/x11/pidgin/pidgin_IRC.pm

```diff
@@ -51,9 +51,10 @@ sub run {
     }
 
     # CTCP Version and warning about scan may appear
-    assert_screen([qw(pidgin-ctcp-version pidgin-irc-sledtesting)]);
+    assert_screen([qw(pidgin-ctcp-version pidgin-ready)]);
     if (match_has_tag('pidgin-ctcp-version')) {
-        wait_screen_change { send_key "ctrl-w"; }    # close it
+        wait_screen_change { send_key "ctrl-w" };    # close it
```

Comment:
no you would not need the `wait_screen_change` in before the next assert_screen but I can merge nevertheless. Hint for next time :)